lib/config/common.ts

```diff
@@ -198,19 +197,19 @@ export interface PackageRule
   extends RenovateSharedConfig,
     UpdateConfig,
     Record<string, any> {
-  paths?: string[];
-  languages?: string[];
-  baseBranchList?: string[];
-  datasources?: string[];
-  depTypeList?: string[];
-  packageNames?: string[];
-  packagePatterns?: string[];
-  excludePackageNames?: string[];
-  excludePackagePatterns?: string[];
+  matchPaths?: string[];
+  matchLanguages?: string[];
+  matchBaseBranches?: string[];
+  matchManagers?: string | string[];
+  matchDatasources?: string[];
+  matchDepTypes?: string[];
+  matchPackageNames?: string[];
+  matchPackagePatterns?: string[];
+  matchNotPackageNames?: string[];
+  matchNotPackagePatterns?: string[];
   matchCurrentVersion?: string | Range;
-  sourceUrlPrefixes?: string[];
-
-  updateTypes?: UpdateType[];
+  matchSourceUrls?: string[];
```

Comment:
Shouldn't this be `matchSourceUrlsPrefixes`? Don't we do prefix match as intended?

Reply:
We do, but I was thinking it was too long of a name. I guess prefixes is better explanatory though. @zharinov can you make it `matchSourceUrlPrefixes` instead?